Services/Dashboard/GlobalScreen/classes/PDLayoutProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+use ILIAS\GlobalScreen\Scope\Layout\Provider\AbstractModificationProvider;
+use ILIAS\GlobalScreen\Scope\Layout\Provider\ModificationProvider;
+use ILIAS\GlobalScreen\ScreenContext\Stack\ContextCollection;
+use ILIAS\GlobalScreen\ScreenContext\Stack\CalledContexts;
+use ILIAS\GlobalScreen\Scope\Layout\Factory\MainBarModification;
+use ILIAS\UI\Component\MainControls\MainBar;
+
+/**
+ * Class PDLayoutProvider
+ *
+ * @author Nils Haagen <nils.haagen@concepts-and-training.de>
+ */
+class PDLayoutProvider extends AbstractModificationProvider implements ModificationProvider
```

Comment:
What does PD standing for?

Reply:
Hi @alex40724 , frankly, I do not know, I simply copied from the existing PDMainBarProvider.